src/BloomBrowserUI/bookEdit/StyleEditor/StyleEditor.js

```diff
@@ -777,18 +777,18 @@ var StyleEditor = (function () {
             if (index > 0)
                 styleName = styleName.substring(0, index);
         }
-        if (this.shouldSetDefaultRule()) {
-            localizationManager.asyncGetText('BookEditor.DefaultForText', 'This formatting is the default for all text boxes with \'{0}\' style', styleName).done(function (translation) {
-                $('#formatCharDesc').html(translation);
-            });
-            return;
-        }
 
         //BL-982 Use language name that appears on text windows
         var iso = $(this.boxBeingEdited).attr('lang');
         var lang = localizationManager.getLanguageName(iso);
         if (!lang)
             lang = iso;
+        if (this.shouldSetDefaultRule()) {
```

Comment:
It's not at all obvious why moving this block helped, so if there is some reason doing it needs to happen after one of the lines above, it would be helpful to comment (even if all you can say is, I don't know why but BLxxxx happens if you do this before YYYY)